+sig/+node/transfererOpCode.m

```diff
@@ -10,8 +10,6 @@
   case {'sig.transfer.mapn' 'sig.transfer.map'}
     if isa(transArg, 'function_handle')
       switch func2str(transArg)
-        case 'numel'
-          code = 30;
```

Comment:
??

Reply:
I was going back and forth between the tests and transfererOpCode/mexnet and wanted the order of the mexOps to be consistent, and it's nice to see it in a sorted ascending order.